core/src/main/java/com/google/googlejavaformat/java/java15/Java15ModifierOrderer.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2021 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.googlejavaformat.java.java15;
+
+import com.google.googlejavaformat.Input.Token;
+import com.google.googlejavaformat.java.ModifierOrderer;
+import javax.lang.model.element.Modifier;
+
+public class Java15ModifierOrderer extends ModifierOrderer {
+
+  @Override
+  protected Modifier asModifier(Token token) {
+    Modifier m = super.asModifier(token);
+    if (m != null) return m;
+    else if (token.getTok().getText().equals("sealed")) return Modifier.valueOf("SEALED");
```

Comment:
What do you think about putting this directly in `ModifierOrderer`? Checking the text of the modifier token and using `Modifier.valueOf` should still compile with earlier language levels. The main reason to have separate classes like `Java14InputAstVisitor` is to allow references APIs only available at the language level, like new AST nodes.

Reply:
Sorry, I should have mentioned my reasoning in my commit.  The thing that speaks against doing it that way is that `Modifier.valueOf` will throw `IllegalArgumentException` if given an unknown name, which will be the case for Java 14 and earlier.  I figured that this was a bit cleaner, as it avoids handling the potential `IllegalArgumentException` and also avoids handling (and dismissing) an exception for Java 15 and later, should there have been a typo in the source,  `"SELAED"`, for example.

Reply:
If there's an invalid modifier it should fail to parse and this code will never be executed, so I think it's fine to have it always try to handle `sealed` and `non-sealed`

Reply:
Ah, of course.